cip-0050/cip-0050.md

```diff
@@ -0,0 +1,63 @@
+## CIP 0050
+
+<pre>
+  CIP: <CIP number, or "?" before being assigned>
+* Layer: <TBD> [TBD see Layer definition cip-XXXX](../../cips/blob/main/cip-XXXX).
+  Title: <CIP title; maximum 44 characters>
+  Author: <list of authors' real names>
+* Discussions-To: <email address>
+* Comments-Summary: <summary tone>
+* Comments-URI: <links to wiki page for comments>
+  Status: <Draft | Active | Proposed | Deferred | Rejected |
+           Withdrawn | Final | Replaced | Obsolete>
+  Type: <Standards Track | Governance | Tokenomics | Process | Informational>
+  Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+  License: <abbreviation for approved license(s)>
+* License-Code: <abbreviation for code under different approved license(s)>
+* Post-History: <dates of postings to [https://lists.sync.global/g/cip-discuss] mailing list, or link to thread in mailing list archive>
+* Requires: <CIP number(s)>
+* Replaces: <CIP number>
+* Superseded-By: <CIP number>
+</pre>
+
+## Abstract
+
+- Allow existing Validator operators to create up to 10 additional Validators for third-party participants.
+- Redirect new individual Validator applicants to approved existing operators.
+- Maintain a controlled, trusted, and orderly network expansion.
+
+## Specification
+
+- Existing Validator operators will be authorized to create up to 10 additional Validators for third-party participants.
+- Third-party applicants wishing to run a Validator will be redirected to an existing operator.
```

Comment:
Does this mean we are not accepting new applicants that want to run their own node anymore?

Reply:
or does it just mean that you can get a node run by an existing validator operator without going through an approval process but the approval process is still there for applicants wanting to run their own node?

Reply:
2nd case. The application is to become your own operator.

Reply:
makes sense, consider folding this comment thread into the text. At least to me it's not clear just from the text that this is what is meant.

Reply:
Good suggestion. Added a bullet and link to the form